### PR TITLE
Update Partner doc with Mobile Assessments

### PR DESCRIPTION
**Pull request by catong**
This PR updates the partner "Building and Running an edX Course" guide and the edX Learner's Guide to reflect the types of assessment that are currently supported in the mobile app. Changes include adding a "mobile-ready" indication to the table of exercises and tools support.
https://openedx.atlassian.net/browse/DOC-2117
### Reviewers

Check the appropriate box after each reviewer finishes and gives :+1:.
- [ ] Subject matter expert: @nasthagiri 
- [x] Subject matter expert: @aleffert 
- [x] Doc team review: @lamagnifica  
- [x] Product review: @explorerleslie 
- FYI: 
### Draft HTML Documentation
- [x] http://draft-br-mobile-assessments.readthedocs.org/en/latest/exercises_tools/create_exercises_and_tools.html#introduction-to-exercises-and-tools
### Post-review
- [ ] Squash commits

### CODE REVIEW

**Comment by lamagnifica** [comment]
:+1: thanks!

**Comment by explorerleslie** [comment]
:+1:

File: en_us/shared/students/SFD_mobile.rst [hunk 1]
```diff
@@ -66,20 +66,6 @@ Courseware Questions
...
-  including readings, homework problems, and exams, use a computer.
-
-.. The following paragraph describes the features of the edX mobile app for Open edX (adds notifications, assessments, discussions)
-.. Alison, DOC-1840, June 2015
-
-.. only:: Open_edX
-
   Not entirely. With the edX mobile app, you can download course videos to
   watch when you do not have an Internet connection. When you have an Internet
   connection, you can also read course announcements and content, participate
```
Comment (nasthagiri) [inline en_us/shared/students/SFD_mobile.rst]: Discussions won't be released at the same time as assessments.  So we should remove "participate in discussions".  And we probably don't need to call it out separately for open edX until it's completely ready to go, right?

**Comment by nasthagiri** [comment]
:+1: lgtm except for the "discussions" comment.

**Comment by aleffert** [comment]
:+1:

**Comment by catong** [comment]
FYI @nasthagiri @explorerleslie @aleffert 
Made edits to clarify access to course discussions on edx.org per Nimisha's review comment.

**Comment by catong** [comment]
Squashed commits; merging when tests pass.

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/572

**Pull request merged**